CONTRIBUTING.md

```diff
@@ -145,6 +145,13 @@ Standard](https://github.com/alleyinteractive/alley-coding-standards) which is a
 modified version of the [WordPress Coding
 Standards](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/).
 
+One exception to this are unit tests which follow Alley Coding Standards but are
+stored in PSR-4-compliant file paths. For example, a class file created with a
```

Comment:
```suggestion
stored in [PSR-4-compliant file paths](https://www.php-fig.org/psr/psr-4/). For example, a class file created with a
```

I don't know if we want to add a link, but it might be helpful